### PR TITLE
Update AWS config comments in spinnaker*.yml

### PR DESCRIPTION
**Pull request by ewiseblatt**
@duftler, @vjames-bmc

### CODE REVIEW

File: config/spinnaker.yml [hunk 1]
```diff
@@ -165,12 +165,12 @@ providers:
     defaultSimpleDBDomain: CLOUD_APPLICATIONS
     primaryCredentials:
       name: default
-      # You can use a standard $HOME/.aws/credentials file instead of providing
-      # these here. If provided here, then start_spinnaker will set environment
-      # variables before starting up the subsystems that interact with AWS.
-      # Do not remove these.
-      access_key_id:
-      secret_key:
+      # The actual credentials are set using a standard AWS client library means
```
Comment (duftler) [inline config/spinnaker.yml]: The word 'means' seems out of place here I think.

File: config/spinnaker.yml [hunk 2]
```diff
@@ -165,12 +165,12 @@ providers:
...
+      # http://docs.aws.amazon.com/AWSSdkDocsJava/latest/DeveloperGuide/credentials.html
+      # Typically this is a $HOME/.aws/credentials file (remember, a standard
+      # spinnaker installation runs as user "spinnaker" whose $HOME is
+      # /home/spinnaker). The primaryCredentials.name will identify which profile
+      # to use (for .aws/credentials)
```
Comment (duftler) [inline config/spinnaker.yml]: Missing terminating period.

**Comment by duftler** [comment]
Seems like this subsumes #767.
LGTM.

**Comment by vjames-bmc** [comment]
@ewiseblatt Thanks. This looks good to me and explains better than #767 . I will close #767

**Pull request merged**